network.go

```diff
@@ -903,6 +903,11 @@ func (n *network) updateSvcRecord(ep *endpoint, localEps []*endpoint, isAdd bool
 			ipv6 = iface.AddressIPv6().IP
 		}
 
+		sb := ep.Sandbox()
```

Comment:
This will work only for bridge networks and not for overlay. Since we are doing the short name alias by default it should work for overly as well.